.goreleaser.yml

```diff
@@ -27,9 +27,6 @@ builds:
       - '386'
       - arm
       - arm64
-    ignore:
-      - goos: darwin
-        goarch: '386'
```

Comment:
Don't you still need to ignore the Apple x86 build?  IIRC no recent version of MacOS supports x86.

Reply:
You're absolutely right - I could not for the life of me remember why we had this ignored. Thanks for that let; going to revert that logic